homeassistant/components/lamarzocco/strings.json

```diff
@@ -109,15 +109,15 @@
         "state": {
           "disabled": "Disabled",
           "prebrew": "Prebrew",
-          "preinfusion": "Preinfusion"
+          "typeb": "Preinfusion"
         }
       },
       "steam_temp_select": {
         "name": "Steam level",
         "state": {
-          "1": "1",
-          "2": "2",
-          "3": "3"
+          "126": "1",
+          "128": "2",
+          "131": "3"
         }
```

Comment:
I mean, if the value still means the same, we should just create a mapping between the two in the integration. Otherwise this is a breaking change, and if the meaning of the variable doesn't change, it's not worth it imo